packages/next-on-pages/templates/_worker.js/utils/cache.ts

```diff
@@ -51,6 +54,9 @@ export async function handleSuspenseCacheRequest(request: Request) {
 			case 'POST': {
 				// Update the value in the cache.
 				const body = await request.json<IncrementalCacheValue>();
+				// Falling back to the cache tags header for Next.js 13.5+
+				body.tags = body.tags ?? body.data.tags ?? request.headers.get(CACHE_TAGS_HEADER)?.split(',') ?? [];
```

Comment:
```suggestion
				body.tags = body.tags ?? body.data.tags ?? request.headers.get(CACHE_TAGS_HEADER)?.split(',')?.filter(Boolean) ?? [];
```
Since the header defaults to `''` (`[CACHE_TAGS_HEADER]: tags?.join(',') || ''`), I think that might mean we could end up with `['']` since we're splitting on `,`, so we should filter that out.